server/data/hmppsAuditClient.test.ts

```diff
@@ -57,7 +57,7 @@ describe('hmppsAuditClient', () => {
       expect(actualMessageBody).toEqual(expectedSqsMessageBody)
 
       const eventTime = Date.parse(actualMessageBody.when)
-      expect(Date.now() - eventTime).toBeLessThan(20)
+      expect(Date.now() - eventTime).toBeLessThan(40)
```

Comment:
```suggestion
      expect(eventTime).not.toBeNaN()
      expect(eventTime).toBeGreaterThan(-1)
      expect(Date.now() - eventTime).toBeLessThan(1000)
```
This is the fix that went into the typescript template project recently, if we just want to use this instead?

Reply:
Updated